app/views/layouts/application.html.haml

```diff
@@ -35,7 +35,7 @@
                   - else
                     = ti("devise.mockdata")
             -# Admin panel
-            - if conf_admin?
+            - if admin?
```

Comment:
Why did we change that, I see the comment @ManuelMoeri left but it was never answered :roll_eyes: